pkg/cmd/openshift-apiserver/openshiftapiserver/configprocessing/audit.go

```diff
@@ -0,0 +1,75 @@
+package configprocessing
+
+import (
+	"io"
+
+	"github.com/golang/glog"
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	auditinternal "k8s.io/apiserver/pkg/apis/audit"
+	auditv1beta1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
+	"k8s.io/apiserver/pkg/audit"
+	auditpolicy "k8s.io/apiserver/pkg/audit/policy"
+	auditlog "k8s.io/apiserver/plugin/pkg/audit/log"
+	auditwebhook "k8s.io/apiserver/plugin/pkg/audit/webhook"
+	pluginwebhook "k8s.io/apiserver/plugin/pkg/audit/webhook"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+)
+
+func GetAuditConfig(auditConfig configapi.AuditConfig) (audit.Backend, auditpolicy.Checker, error) {
```

Comment:
have you checked what is different here compared to upstream?

Reply:
> have you checked what is different here compared to upstream?

I have not